### PR TITLE
Add a note about the point-in-time concern with VS and the CLI

### PR DESCRIPTION
**Pull request by BillWagner**
The .NET SDK CLI does not yet include the rename from NullableContextOptions to Nullable.

See https://github.com/dotnet/docs/pull/12497#issuecomment-495242895

/cc @rainersigwald

### CODE REVIEW

**Review by rainersigwald — APPROVED** [review]
LGTM but I've been talking offline with @jaredpar about this so he should weigh in.

File: docs/csharp/nullable-references.md
Comment (rpetrusha) [inline docs/csharp/nullable-references.md]: I think I'd label this Important rather than Note for greater visibility.

**Review by rpetrusha — APPROVED** [review]
LGTM, @BillWagner. Left one comment for you to consider that applies to all three of your changes. You can merge when you're ready.

**Review by Thraka — APPROVED** [review]
LGTM too 😄

**Review by Thraka — CHANGES_REQUESTED** [review]
~Actually can we hold off for a second? I want to test something.~

**Review by Thraka — APPROVED** [review]

**Review by drewnoakes — APPROVED** [review]

**Pull request merged**
